### ISSUE TITLE
Create new ADRs in the "Proposed" state by default

### ISSUE THREAD

**Issue by npryce** [npryce/adr-tools#61]
Currently a new ADR is created in the Accepted state by default.

I suggest it is better to create new ADRs in the Proposed state.  ADRs are meant to be discussed and agreed within a team. When team members are not all colocated, discussion can occur around or within the ADR document itself, until the team agree that the ADR is ready to be marked as Accepted.

This would also encourage teams to record proposed architecture decisions that are discussed and rejected, along with the reasons for rejection.

**Comment by retnuh** [comment]
Makes sense to me.  Another place we've seen discussion take place is on the PR for the ADR itself, but even then there was a "Proposal" phase that gets committed and a follow up with the relevant points from the discussion.

**Comment by npryce** [comment]
This is not as easy as I first thought.  Some commands treat the status "Accepted" as a special case (e.g. `adr-new` with the `-s` option).  They would have to be changed to support Proposed and Accepted, or be more general (e.g. assume that the first non-blank line of the Status section is the status, handle the case when that is a link, etc.

**Comment by simkimsia** [comment]
Out of curiosity, how many states are there for ADR assumed by this tool? Btw, I like the idea of having a Proposed state by default.

**Comment by npryce** [comment]
At the moment it’s free-form apart from apart from Accepted and Superceded

**Comment by PeterMTaylor** [comment]
This is a good idea. I guess for a use case example would be that all incoming work and their definitions for discussion for the next period/cycle of time would show proposed status which triggers discussion. Once the conversations are over, majority of the items becomes approved or rejected, any left over in proposed status shows up in the next period/cycle.
